### DISCUSSION TITLE
error: building openssl:x64-windows failed with: BUILD_FAILED

### DISCUSSION THREAD

**Discussion by A-hua-code** [microsoft/vcpkg#41764] [Q&A]
[issue_body.md](https://github.com/user-attachments/files/17514417/issue_body.md)
![error](https://github.com/user-attachments/assets/86452ce7-9aef-4145-bdf2-095a22afbb76)

**Comment by greenozon** [comment]
Please attach logs

**Reply by 321915514** [reply]
openssl build error: 
Failure!  build file wasn't produced.
Please read INSTALL.md and associated NOTES-* files.  You may also have to
look over your available compiler tool chain or change your configuration.

NASM not found - make sure it's installed and available on %PATH%

but nasm -v is right
